service-worker.js

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = 1;
+
+toolbox.cache.name = "Babel-Cache-" + VERSION;
+
+var preCachedRessources = [
+  {% for page in site.pages %}
+  '{{ page.url }}',
+  {% endfor %}
+];
+
+toolbox.precache(preCachedRessources);
+
+toolbox.router.get('/*', toolbox.cacheFirst);
```

Comment:
I wanted an offline documentation. This caching strategy might indeed not be the best approach.